CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.2.1
+ - new feature: Add new option "push_map_as_event_on_timeout" so that when events time out they can be yielded as new events
+ - new feature: Add new option "timeout_code" which takes the event and executes code on it. This works for "push_map_as_event_on_timeout" as well as "push_previous_map_as_event"
+
```

Comment:
Given that it is clearly a new feature, the new release will be version 2.3.0.
I keep the last digit for bug fixes/minor changes.
=> Can you put 2.3.0 in changelog ?